lib/bounces.js

```diff
@@ -120,14 +120,7 @@ module.exports = function (log, error) {
 
           // Flow events have a specific event and structure that must be emitted.
           // Ref `gather` in https://github.com/mozilla/fxa-auth-server/blob/master/lib/metrics/context.js
-          const flowEventInfo = {
-            event: eventName,
-            time: currentTime,
-            flow_id: flowId,
-            flow_time: elapsedTime
-          }
-
-          log.flowEvent(flowEventInfo)
```

Comment:
So, in train-77 (current prod), the above line is `log.info(flowEventInfo)`. So @pb fixed this to be `log.flowEvent(flowEventInfo)` already? (if that's the proper use of `log.flowEvent`).